section-09-differential-testing/model-package/tests/test_prediction.py

```diff
@@ -0,0 +1,22 @@
+import math
+
+import numpy as np
+
+from regression_model.predict import make_prediction
+
+
+def test_make_prediction(sample_input_data):
+    # Given
+    expected_first_prediction_value = 113422
+    expected_no_predictions = 1449
+
+    # When
+    result = make_prediction(input_data=sample_input_data)
+
+    # Then
+    predictions = result.get("predictions")
+    assert isinstance(predictions, list)
+    assert isinstance(predictions[0], np.float64)
+    assert result.get("errors") is None
```

Comment:
all of this looks very similar to that found in the file test_model_accuracy. Is there a way of not repeating?

Reply:
One is for capturing the predictions, so different purposes. I'd leave it.